### ISSUE TITLE
[Admin] Deceased Date required when adding a new patient

### ISSUE THREAD

**Issue by benCoder01** [CombiRWTH/medical-staff-assessment#77]
When adding a new patient in the django admin interface, the deceased date is required, obviously this is not correct.

**Comment by benCoder01** [comment]
And somehow i can not create a patient with that information
![image](https://github.com/user-attachments/assets/c55aef30-0661-474b-a1b4-8154f3324632)

**Comment by Ishan-Jadhav** [comment]
The first issue can be addressed by adding `null=True` in the model definition of the Patient.
```
class Patient(models.Model):
     """Patients in the database."""

    id = models.IntegerField(primary_key=True)
    first_name = models.CharField(max_length=100)
    last_name = models.CharField(max_length=100)
    date_of_birth = models.DateField()
    weight = models.FloatField()
    height = models.FloatField()
    deceased_date = models.DateField(null=True)  # Date patient passed away

    def __str__(self):
        return f"{self.first_name} {self.last_name}"
```

The model definition does not include a station_id column. Could you clarify how the second issue related to station_id is occurring?

**Comment by benCoder01** [comment]
Yeah the screenshot i posted, was my mistake (undeleted databse volume), I delteted that comment . But if you want, you can try to solve that issue, i think with null=True, that could work

**Comment by EricPossato** [comment]
If this issue isn't done already, could I get it assigned? I would like to solve it

**Comment by EricPossato** [comment]
Since it's a very simple fix, I added a PR already to save some time, hope thats alright.
